erpnext/stock/doctype/item/item.js

```diff
@@ -587,25 +587,37 @@ $.extend(erpnext.item, {
 			me.multiple_variant_dialog = new frappe.ui.Dialog({
 				title: __("Select Attribute Values"),
 				fields: [
+					frm.doc.image
+						? {
+								fieldtype: "Check",
+								label: __("Create a variant with the template image."),
+								fieldname: "use_same_image",
```

Comment:
Please change the variable name use_same_image -> use_template_image

Reply:
Done 👍